### ISSUE TITLE
[Objectron]ScreenOrientation auto rotate problem.

### ISSUE THREAD

**Issue by RMiTiG** [homuler/MediaPipeUnityPlugin#212]
The Objectron sample has supported LandscapeRight mode now, I just need to modify the camera to fill screen with WebCamScreen.
For Portrait mode, I have tried to rotate the WebCamScreen or MainCamera reference this [page](https://github.com/homuler/MediaPipeUnityPlugin/issues/131),but the  bouding box didn't match to the shoe.
So I don't rotate them,I add a rotation_mode option as 2 (rotate 90 degree in clockwise)in ImageTransformationCalculator.
Both in object_detection_3d_android and object_detection_3d_gpu graph:
```
# Flip vertically because the input image is aligned from bottom-left to top-right.
node: {
  calculator: "ImageTransformationCalculator"
  input_stream: "IMAGE_GPU:throttled_input_video_gpu"
  output_stream: "IMAGE_GPU:transformed_input_video"
  node_options: {
    [type.googleapis.com/mediapipe.ImageTransformationCalculatorOptions] {
      flip_vertically: true
      rotation_mode: 2
    }
  }
}
```
This can detects shoe correctly both in ios and android when orientation mode is Portrait ,but it's output_video's aspect ratio is 4:3(width:height) although output_stream is 3:4 in ImageCroppingCalculator.Seem like that focal length is half smaller. 
There is same problem if I add rotation option as 4.712388981(rotate 270 degree in counter-clockwise) in ImageCroppingCalculator instead of changing option in ImageTransformationCalculator.
And it looks strange if I set WebCamScreen's scale x/z as 3/4, or set input_stream's width/height  as 480/640.
How can I receive output_video in 3:4?For this, I have tried to remove the ImageCroppingCalculator and set ImageTransformationCalculator's output to ObjectronGpuSubgraph's input directly(WebCamScreen's scale x/z is 3/4), it work well in android(1080x2408), just is exactly what I want.But it looks strange in ios(1170x2532):
![test](https://user-images.githubusercontent.com/39582528/124239372-32ba4e00-db4c-11eb-81a9-14c425bedb1f.jpg)

Otherwise, I try to change the input_stream's width/height to 1280/960 because the image looks very blurry.That is useful for Android, but console this error in ios: SetPixels32 called with invalid number of pixels in the array.
There is no idea with it.Hope for help.

**Comment by emreboyraz0** [comment]
MainCamera : Rotation Z = 90 
MainCamera :Scale X=-1 Y=-1 Z=1

Just do this settings then you can use in portrait mode. 
I am using portrait mode with this settings and everything is okey for me.

**Comment by RMiTiG** [comment]
@emreboyraz0 Thanks for reply, but it's detection always go wrong if I rotate the camera.Because it alway judge the left side as bottom side I guess.

**Comment by emreboyraz0** [comment]
I did not change 640x480 in webCamScreenController. Also did not change object_detection_3d_android and object_detection_3d_gpu. 

Just change
MainCamera : Rotation Z = 90
MainCamera :Scale X=-1 Y=-1 Z=1

And 
WebCamScreen : Rotation X=90, Y=180, Z=0
WebCamScreen : Scale X=4, Y=1 Z=3

And now i can using app in portrait mode in Android and IOS without any problem. And not pixelated

**Comment by RMiTiG** [comment]
@emreboyraz0 You are right,it detects better when I reset the WebCamScreen size.Maybe the amount of pixel influence the calculator, I will find other ways after.Thanks a lot.

Otherwise, the strange problem may be caused by the IMAGE->IMAGE_GPU, reference https://github.com/google/mediapipe/issues/704#issuecomment-625392448, I think it's unavoidable or we can use cpu instead of gpu in ios

**Comment by nenenkosi** [comment]
@emreboyraz0 and @RMiTiG I have been reading and following all the issues and I see that you guys have managed have to get the 3D shoe object_detection on both Android and IOS working and it’s been giving me problems so can I  please get your Twitter usernames so I can follow you’ll so I get assistance from you’ll or can you please share a GitHub link to the work project on the 3D shoe object_detection coz I really need assistance this is my email nenenkosi@gmail.com

**Comment by RMiTiG** [comment]
@nenenkosi Hi,sorry that I don't active on Twitter, and my work there is some difference to try-on and it is merged into other private project that I can not share yet.
But I can share you my experences.This MediaPipeUnityPlugin has help us a lot to build and show in Unity, it's really a great work.And it also do some special work that you can see in the code.And make it has some difference with google/Mediapipe.For working both in Android and iOS in unity, you can see this https://github.com/homuler/MediaPipeUnityPlugin/issues/182, I have updated some method in it.Hope it useful for you.

**Comment by nenenkosi** [comment]
@RMiTiG Thanks for your quick reply. I understand you can not share the work project the only thing I was asking for was a start-up project that solves the few main issues coz I know you have solved them e.g am not able to distinguish between the right and left shoe and am getting a dark render on IOS I know you have solved all the issues on #182 so I was just asking for a simple project with all the issues solved any kind of help will be highly appreciated.
And can I have your email or any other means of communication so you can guide me through the project so I don’t keep adding it here .
Thanks in advance
